Baekjun16235.java

```diff
@@ -0,0 +1,147 @@
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.ArrayDeque;
+import java.util.PriorityQueue;
+import java.util.StringTokenizer;
+
+public class Baekjun16235 {
+	private static int[][] A;
+	private static int[][] G;
+	private static PriorityQueue<Tree> trees;
+	private static ArrayDeque<Tree> trees_alive = new ArrayDeque<>();
+	private static ArrayDeque<Tree> trees_dead = new ArrayDeque<>();
+	private static int N;
+
+	public static void main(String[] args) throws Exception {
+		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+		StringTokenizer st = new StringTokenizer(br.readLine());
+		N = Integer.parseInt(st.nextToken()); // 땅 넓이
+		int M = Integer.parseInt(st.nextToken()); // 나무 수
+		int K = Integer.parseInt(st.nextToken()); // 년
+		A = new int[N][N];
+		G = new int[N][N];
+		for (int i = 0; i < N; i++) {
+			st = new StringTokenizer(br.readLine());
+			for (int j = 0; j < N; j++) {
+				A[i][j]=Integer.parseInt(st.nextToken());
+				G[i][j]=5;
+			}
+		}
+		trees = new PriorityQueue<Tree>(); // 어린 나무부터 양분을 먹기위한 우선순위 큐
+		for (int i = 0; i < M; i++) {
+			st = new StringTokenizer(br.readLine());
+			int x = Integer.parseInt(st.nextToken())-1;
+			int y = Integer.parseInt(st.nextToken())-1;
+			int age = Integer.parseInt(st.nextToken());
+			trees.add(new Tree(x,y,age));
+		}
+		
+		
+		for (int i = 0; i < K; i++) { // K년이 지난다
+			one_year_later();
+		}
+		
+		System.out.println(trees.size());
+	}
+
+	private static void one_year_later() { // 사계절이 지난다
+		Spring();
+		Summer();
+		Fall();
+		Winter();
+	}
+
+	private static void Spring() {

```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.